modules/notes/controllers/updateNotes.js

```diff
@@ -0,0 +1,12 @@
+import { updateNote } from "../../../db/providers/notesProvider.js";
+import logger from "../../../utils/logger.js";
+
+
+ export  const updateOneNote = async(req, res) => {
+    try {
+      const noteUpdate = await updateNote(req.body);
```

Comment:
Ooops!
What we will do in case I will send any kind of hack scripts in body? You will send it to the DB!!! I can potentially destroy it.
Separate all the required info from body in controller and send to the provider filtered data.